tipsy-bot/tipsy.py

```diff
@@ -87,13 +82,14 @@ async def person_detect(detector: VisionClient, sensors: list[Sensor], base: Bas
 async def main():
     robot = await connect()
     base = Base.from_robot(robot, base_name)
-    sensors = [Sensor.from_robot(robot, sensor_name) for sensor_name in sensor_names]
+    sensors_svc = SensorsClient.from_robot(robot=robot, name="builtin")
```

Comment:
Since this defaults to "builtin" as the `name`, should we set it at all? Otherwise, it seems like it could be configurable. 

```suggestion
    sensors_svc = SensorsClient.from_robot(robot=robot)
```

Reply:
I think the name is required possibly. Haven't tested though. Hazal if you want to test this when you run the code please do

Reply:
When looking at the source code, the default value is "builtin"

Reply:
I didn't see that on the stub but I'll trust you 😂